src/components/Editor/Menu/Fixed/index.jsx

```diff
@@ -44,10 +44,46 @@ const Fixed = ({
   setIsEmojiPickerActive,
   children,
 }) => {
-  const { t } = useTranslation();
+  const [focusedButtonIndex, setFocusedButtonIndex] = useState(-1);
   const [isMenuExpanded, setIsMenuExpanded] = useState(false);
   const [isEmbedModalOpen, setIsEmbedModalOpen] = useState(false);
 
+  const menuRef = useRef(null);
+
+  const { t } = useTranslation();
+
+  const handleArrowNavigation = event => {
+    if (event.key === "ArrowRight") {
+      event.preventDefault();
+      setFocusedButtonIndex(prevIndex =>
+        prevIndex === -1 ? prevIndex + 2 : (prevIndex + 1) % menuButtons.length
+      );
```

Comment:
```suggestion
      setFocusedButtonIndex(prevIndex => (prevIndex + 1) % menuButtons.length)
```

Can we get away with this implementation? For this to work, the default value in the `useState` should be 0.

Reply:
I will work on that, This might be possible once i update the useEffect.